frontend/src/console/components/CoursesManage/displayMessages.ts

```diff
@@ -0,0 +1,35 @@
+const messages = {
+  title: {
+    defaultMessage: 'Courses (Studio)',
+    description: 'The component title'
+  },
+  paragraph_01: {
```

Comment:
Rename this to a more descriptive title.

Reply:
Applied.